docs/unified-test-documentation/dasharo-security/20F-me-neuter.md

```diff
@@ -13,7 +13,41 @@
 1. Proceed with the
     [Generic test setup: OS boot from disk](../../generic-test-setup#os-boot-from-disk).
 
-## MNE001.001 Intel ME mode option is available and has the correct default state
+## MNE001.001 Check Intel ME version (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to check Intel ME version, should be the same as in
```

Comment:
@Psotas We strive to maintain a unified test documentation structure. So, in my opinion, it would be good to start this section in the following way: 

```
This test aims to verify that <task here>
```

Reply:
I change test description to:
This test aims to verify the Intel ME version, should be the same as in the release notes.